tests/MSBuildDeviceIntegration/Tests/InstallAndRunTests.cs

```diff
@@ -1130,5 +1130,41 @@ public void FixLegacyResourceDesignerStep ([Values (true, false)] bool isRelease
 				Assert.IsTrue (didLaunch, "Activity should have started.");
 			}
 		}
+
+		[Test]
+		public void MicrosoftIntune ([Values (false, true)] bool isRelease)
+		{
+			if (!Builder.UseDotNet)
+				Assert.Ignore ("Test is only valid in .NET 8+");
```

Comment:
We can probably remove this, unless it's going to be CPd to any classic branches?